### PR TITLE
Activate: Login button swap experiment

### PR DESCRIPTION
**Pull request by 0nko**
This PR adds a new login improvement experiment, which swaps the login CTA buttons (`Continue with WordPress.com` and `Enter your store address`).

image

**To test:**

1. Comment out the contents of the `FirebaseRemoteConfigRepository::fetchRemoteConfig()` and change the `LoginButtonSwapVariant.CONTROL.name` to `LoginButtonSwapVariant.SWAPPED.name` of `defaultValues` in the same class
2. Start the app (skip the prologue carousel if it's shown) and get to the login prologue screen
3. Verify that `Continue with WordPress.com` is the primary button and `Enter your store address` is a secondary button
4. Change the `LoginButtonSwapVariant.SWAPPED.name` to `LoginButtonSwapVariant.CONTROL.name` 
5. Run the app again and verify that `Enter your store address` is the primary button and `Continue with WordPress.com` is a secondary button

### CODE REVIEW

File: WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/LoginActivity.kt [hunk 1]
```diff
@@ -237,7 +239,7 @@ class LoginActivity :
     private fun showMagicLinkInterceptFragment(authToken: String) {
         val fragment = MagicLinkInterceptFragment.newInstance(authToken)
         supportFragmentManager.beginTransaction()
-            .replace(R.id.fragment_container, fragment, LoginPrologueFragment.TAG)
+            .replace(R.id.fragment_container, fragment, MagicLinkInterceptFragment.TAG)
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/ui/login/LoginActivity.kt:242]: Good catch!

File: WooCommerce/src/main/res/layout-land/fragment_login_prologue_swapped.xml [hunk 2]
```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (JorgeMucientes) [inline WooCommerce/src/main/res/layout-land/fragment_login_prologue_swapped.xml:1]: I wonder if you really need to replicate all these layouts for just swapping the buttons. No need to change but wouldn't it be more simple to just update the buttons text and onClick listener from `LoginPrologueFragment` depending on the experiment value?

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
Works well and code look good. I just left a minor suggestion, but since we are tight on schedule I won't block this PR for that. 
Feel free to merge.

File: WooCommerce/src/main/res/layout-land/fragment_login_prologue_swapped.xml (hunk 2 above)
Comment (0nko) [reply WooCommerce/src/main/res/layout-land/fragment_login_prologue_swapped.xml:1]: Yeah, I was thinking of doing it that way but in the end I decided to add a new layout. After the end of the experiment all we'll need to do is remove the one we don't need.

**Comment by 0nko** [comment]
Thanks for the review, @JorgeMucientes! Since it's already working I'd say let's just ship it as it is.

**Pull request merged**